Source: written by Pierre Lejeune, from Exherbo
Upstream: yes, http://cvs.rutgers.edu/cgi-bin/viewvc.cgi/trunk/source-patches/gkrellm.patch?view=co
Reason: Help find symbols of gmodule-2.0

--- a/src/Makefile	2010-10-02 18:21:01.000000000 +0200
+++ b/src/Makefile	2012-04-08 19:47:49.329974443 +0200
@@ -68,8 +68,8 @@
 
 GKRELLM_INCLUDES = gkrellm.h gkrellm-public-proto.h $(SHARED_PATH)/log.h
 
-PKG_INCLUDE = `$(PKG_CONFIG) --cflags gtk+-2.0 gthread-2.0`
-PKG_LIB = `$(PKG_CONFIG) --libs gtk+-2.0 gthread-2.0`
+PKG_INCLUDE = `$(PKG_CONFIG) --cflags gtk+-2.0 gthread-2.0 gmodule-2.0`
+PKG_LIB = `$(PKG_CONFIG) --libs gtk+-2.0 gthread-2.0 gmodule-2.0`
 
 FLAGS = -O2 -I.. -I$(SHARED_PATH) $(PKG_INCLUDE) $(GTOP_INCLUDE) $(PTHREAD_INC)\
  ${SSL_INCLUDE} ${NTLM_INCLUDE} -DGKRELLM_CLIENT
